jenkins/e2e-image/e2e-runner.sh

```diff
@@ -84,6 +84,54 @@ function running_in_docker() {
     grep -q docker /proc/self/cgroup
 }
 
+### FUNCTIONS TO SUPPORT get-kube.sh ###
+
+# Moves local tarballs into a local path suitable for get-kube.sh, and sets
+# KUBERNETES_RELEASE and KUBERNETES_RELEASE_URL accordingly.
+function set_release_vars_from_local_gcs_stage() {
```

Comment:
I should probably move most of `set_release_vars_from_gcs` to `get-kube.sh`, and possibly also write some unit tests for `get-kube.sh`. that is orthogonal to the intent of this PR, which is blocking my ability to pull out the client/server binaries from `kubernetes.tar.gz`.